app/src/main/java/com/luc4n3x/levyra/ui/LevyraApp.kt

```
@@ -1308,6 +1313,37 @@ fun LevyraApp(viewModel: LevyraViewModel, isInPictureInPicture: Boolean = false)
+                !showLanguageRestartDialog
+            PredictiveBackHandler(enabled = playerPredictiveBackEnabled) { backProgress ->
+                val startExpansion = playerExpansion.value
+                try {
+                    backProgress.collect { backEvent ->
+                        playerExpansion.snapTo(
+                            playerPredictiveBackExpansion(startExpansion, backEvent.progress)
+                        )
+                    }
+                    playerExpansion.animateTo(0f, spring(dampingRatio = 0.86f, stiffness = 430f))
+                    if (state.selectedTab == LevyraTab.Player) viewModel.selectTab(backgroundTab)
+                } catch (cancelled: CancellationException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)